swim/ping_request_sender.go

```diff
@@ -82,6 +82,9 @@ func (p *pingRequestSender) SendPingRequest() (*pingResponse, error) {
 	var res pingResponse
 	select {
 	case err := <-p.MakeCall(ctx, &res):
+		if err == nil {
+			p.node.memberlist.Update(res.Changes)
```

Comment:
Is this an unrelated bug that you discovered in the course of this change, i.e. that we aren't applying changes gossiped in ping reqs?  Should this be a separate diff with its own tests?